lib/bn.js

```diff
@@ -1021,6 +1237,12 @@ BN.prototype.mul = function mul(num) {
   return this.mulTo(num, out);
 };
 
+// Multiply employing FFT
+BN.prototype.mulf = function mulf(num) {
+  var out = new BN(null);
+  return jumboMulTo(this, num, out);
```

Comment:
@alexeykudinkin this does not really affect anything except that it violates the current design of these methods. I don't see anything bad about it, and clearly see some performance-wise pros.